examples/demo.rs

```diff
@@ -386,9 +409,21 @@ fn draw(
 }
 
 fn main() {
-    tracy_client::Client::start();
     //env_logger::init_from_env(env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "warn"));
```

Comment:
plz don't remove the tracy client :)
That said, it was missing a feature flag conditional!

Reply:
I actually moved that to just above `GpuProfiler::new_with_tracy_client` on line 152. But I guess that change is kinda not related to this PR, so I'll move it back and just add the conditional